libs/auth/cache/cache.go

```diff
@@ -9,6 +9,7 @@ import (
 type TokenCache interface {
 	Store(key string, t *oauth2.Token) error
 	Lookup(key string) (*oauth2.Token, error)
+	DeleteKey(key string) error
```

Comment:
The other functions are unqualified as well.

```suggestion
	Delete(key string) error
```

Reply:
Implemented in commit 37067ef93391e4e8da6b5b71f7d2be9ad1622883